app/graphql/resolvers/photo_create_resolver.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+module Resolvers
+  class PhotoCreateResolver < Resolvers::Base
+    argument :imageable_id, GraphQL::Types::ID, required: true
+    argument :imageable_type, String, 'Can have value Freak or Project, depending where the photo is assigned',
+             required: true
+    argument :uri, Types::Url, required: true
```

Comment:
We will need to add validations for these arguments. Currently someone can add anything in the fields. Ticket for another time.